skafka/src/main/scala/com/evolutiongaming/skafka/ClientMetric.scala

```diff
@@ -0,0 +1,9 @@
+package com.evolutiongaming.skafka
+
+final case class ClientMetric[F[_]](
+  name: String,
+  group: String,
+  description: String,
+  tags: Map[String, String],
+  value: F[AnyRef]
```

Comment:
could you please explain this a bit?

Reply:
but why this is `AnyRef` ?

Reply:
It's sad, but a metric value could be anything. For example, there are some metrics, related to a version, so they are String. I don't see a big problem here as the value will be probably matched anyway to figure out if it's number

Reply:
it's indeed sad…